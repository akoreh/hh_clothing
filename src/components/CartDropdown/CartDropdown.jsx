import React from 'react';

import {CartDropdownContainer, EmptyMessageContainer, CartItemsContainer} from './CartDropdown.styles';

const CartDropdown = () => (
    <CartDropdownContainer>
        <CartItemsContainer>
            <EmptyMessageContainer>Your cart is empty</EmptyMessageContainer>
        </CartItemsContainer>
    </CartDropdownContainer>
);

export default CartDropdown;