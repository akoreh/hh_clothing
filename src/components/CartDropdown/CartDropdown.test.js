import React from 'react';

import { shallow } from 'enzyme';

import {CartDropdown} from './CartDropdown';

describe('CartDropdown Component', () => {
    let wrapper;

    beforeEach(() => {
        wrapper = shallow(<CartDropdown />);
    });

    it('should render CartDropdown component', ()=> {
        expect(wrapper).toMatchSnapshot();
    });
});