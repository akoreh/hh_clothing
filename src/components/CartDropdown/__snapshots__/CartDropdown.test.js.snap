// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CartDropdown Component should render CartDropdown component 1`] = `ShallowWrapper {}`;
